raiden/api/v1/resources.py

```diff
@@ -0,0 +1,113 @@
+from webargs.flaskparser import use_args, use_kwargs, parser
+from flask_restful import Resource
+from flask import Blueprint
+from pyethapp.jsonrpc import address_encoder
+from raiden.api.v1.encoding import ChannelRequestSchema, AddressField
+
+
+def create_blueprint():
+    # Take a look at this SO question on hints how to organize versioned
+    # API with flask:  http://stackoverflow.com/questions/28795561/support-multiple-api-versions-in-flask#28797512
+    return Blueprint('v1_resources', __name__)
+
+
+class BaseResource(Resource):
```

Comment:
In the new changes I added the proper `super()` call but witout args, since `object` takes no args. In addition I am using this class to properly populate the `rest_api` object of every resourse using the forwarded class kwargs.